code/csv_test.py

```diff
@@ -0,0 +1,19 @@
+#testing writing to CSV files. making sure that we can write fast enough.
+
+import time
+
+f = open("test.txt", "w+")#write to file, if not there, create it.

```

Comment:
```suggestion
f = open("%s-test.txt" % time.time(), "w+")#write to file, if not there, create it.
```
I don't have a strong opinion about this, but you could keep all of your test files separate by creating them with a timestamp.

Reply:
That sounds like a good idea to do. The 'test.txt' was simply temporary. I was planning on implementing something to differentiate them, and this seems like as good an idea as any. I think that I'll implement that in the main files. The file you linked to was a test file for testing if python could write to a file fast enough. If you don't think that's something we need to keep around, the file doesn't need to be kept.

Reply:
Oh ok that makes sense, yeah it's fine to keep it. Maybe when you start writing the main code move everything that you're just using for testing/demos into a separate folder in the repo just to keep it separate from the real code.

Reply:
Sure thing!
I'll do that once the main code starts getting written.